src/cephadm/cephadm

```diff
@@ -6519,6 +6519,16 @@ def command_rm_cluster(ctx):
                 if os.path.exists(files[n]):
                     os.remove(files[n])
 
+            for fname in glob('/var/log/ceph/cephadm.log*'):
+                os.remove(fname)
+            # cleanup remaining ceph directories
+            ceph_dirs = [f'/run/ceph/{ctx.fsid}',
+                         f'/tmp/var/lib/ceph/{ctx.fsid}',
+                         f'/var/log/ceph/{ctx.fsid}',
+                         f'/var/lib/ceph/{ctx.fsid}',
```

Comment:
`/var/log/ceph` and `/var/lib/ceph/` typically corrrelate to the "log dir" and "data dir" for cephadm (they are the default values that will be set for `ctx.log_dir` and `ctx.data_dir`). It looks like we're already taking care of removing them (log dir: https://github.com/ceph/ceph/blob/master/src/cephadm/cephadm#L6506, data dir: https://github.com/ceph/ceph/blob/master/src/cephadm/cephadm#L6502) and it's good to use the ctx value rather than hardcoded ones anyhow in case somebody is actually using a custom location.

Also, we need to be careful about ignoring the `--keep-logs` flag. Logs need to be kept if this is set when `rm-cluster` runs.

One of the linked trackers does seem to indicate we're failing to remove `/var/log/ceph/<fsid>` but if that's the case we should probably fix it where we're already attempting the removal rather than in a new spot. When I tested locally it seemed to be getting removed. Is it sticking around for you?

Reply:
I retested the code but didn't see it there so maybe I just included it the first time by mistake. The new code only remove the direcotries: ```[f'/run/ceph/{ctx.fsid}', f'/tmp/var/lib/ceph/{ctx.fsid}', f'/var/run/ceph/{ctx.fsid}']```